crates/types/src/config/bifrost.rs

```diff
@@ -237,6 +237,13 @@ pub struct ReplicatedLogletOptions {
     ///
     /// Retry policy for log server RPCs
     pub log_server_retry_policy: RetryPolicy,
+
+    /// records_cache_memory_budget_bytes
+    ///
+    /// Optional size of records cache in bytes.
+    /// If set to None, records cache will be disabled.
+    /// Defaults: 20M
+    pub records_cache_memory_budget_bytes: Option<usize>,
```

Comment:
```suggestion
    pub record_cache_memory_size: Option<usize>,
```

Reply:
Can you look at common.rs's configuration for inspiration? For configuration keys that accept bytes, we have some helpers to allow setting those values from human-friendly strings. A good example is `rocksdb_total_memory_size`